data_catalog_backend/routes/admin/resource_routes.py

```diff
@@ -45,3 +72,371 @@ async def add_resource(
         raise HTTPException(
             status_code=500, detail=f"Error creating resource: {str(e)}"
         )
+
```

Comment:
With all these new endpoints, the api changes dramatically from what it is in the main branch.
Have you checked this with Sander, and he is aware of the changes required to the script?